docs/requirements.md

```diff
@@ -1,28 +1,35 @@
 
-The second requirement is necessary because ScalarDB does not work on some Cassandra-compatible databases, such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/), that do not support LWTs. This is because the Consensus Commit transaction manager relies on the linearizable operations of underlying databases to make transactions serializable.
+ScalarDB provides only the atomicity and isolation properties of ACID and requests the underlying databases to provide durability. Although you can specify `periodic`, we do not recommend doing so unless you know exactly what you are doing.
 
-If the above requirements are met, storage operations with `LINEARIZABLE` can provide linearizablity and transaction operations with `SERIALIZABLE` can provide strict serializability.
+### Confirm that the Cassandra-compatible database supports lightweight transactions (LWTs)
 
-## JDBC databases
+You must use a Cassandra-compatible database that supports LWTs.
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code with the `Operation.withConsistency()` method, and the consistency level depends on the setup of your JDBC database.
-For example, when you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas.
-On the other hand, when you perform all operations against a single master instance, the consistency will be linearizable.
-We recommend performing all operations/transactions against a single master instance (so that you can achieve linearizable) if you want to avoid caring consistency issues in your applications. Note that you can still use a read replica as a backup and standby even if you follow the recommendation.
+ScalarDB does not work on some Cassandra-compatible databases that do not support LWTs, such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/). This is because the Consensus Commit transaction manager relies on the linearizable operations of underlying databases to make transactions serializable.
 
-If you strongly want to perform operations/transactions on read replicas, you can configure your application to perform read-write-mixed transactions against a master instance and read-only transactions against read replicas.
-The resulting schedule would not be strict serializable (linearizable and serializable) but would be serializable.
+## CosmosDB database requirements
+
+In your Azure CosmosDB account, you must set the **default consistency level** to **Strong**.
+
+For instructions on how to configure this setting, see the official documentation at [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level).
```

Comment:
@brfrn169 I added the following according to @feeblefakie's comment in https://github.com/scalar-labs/scalardb/pull/1020/files#r1303994693.

Is there any more information that we can add here, such as why **Strong** needs to be configured?

Reply:
We require **Strong** consistency in CosmosDB because the ScalarDB transaction protocol (Consensus Commit) needs linearizable reads. The **Strong** consistency level provides this linearizability guarantee:
https://learn.microsoft.com/en-us/azure/cosmos-db/consistency-levels#strong-consistency

Reply:
@brfrn169 Thanks for the extra details! I've added a couple extra sentences to explain why this setting is necessary.